great-mystery-great-whore/original/025-chapter-24.adoc

```diff
@@ -278,8 +278,7 @@ or steeple-houses to vent their doctrine,`' etc.
 
 [.discourse-part]
 A+++.+++ These are the marks they give to us,
-and to prove us according to the title of their book,
-'`Perfect Pharisees`'. The apostles preached as they went up and down, in towns, markets,
+and to prove us according to the title of their book, [.book-title]#Perfect Pharisees.# The apostles preached as they went up and down, in towns, markets,
```

Comment:
I wasn't sure if I should make this into a book title or not.

Reply:
Yeah, a bit confusing... I think I lean towards keeping it single quotes how it was before, now that I've thought about it a bit.